### PR TITLE
Kuttl v0.2.0

### PR DESCRIPTION
**Pull request by kensipe**

### CODE REVIEW

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

This pull request **does not** seem to be a straightforward version bump. I'll have a human review this.

_Why wasn't this detected as a plugin version bump:_

>file plugins/kuttl.yaml is not a straightforward version bump: diff line 8 unrecognized for version bumps: [+    files:]

File: plugins/kuttl.yaml [hunk 1]
```diff
@@ -18,27 +18,39 @@ spec:
...
         os: "linux"
         arch: "amd64"
-    uri: https://github.com/kudobuilder/kuttl/releases/download/v0.1.0/kuttl_0.1.0_linux_x86_64.tar.gz
-    sha256: "4043a3052a0f5b6c66b80fb1ad8dcfba10cb1e3a7f40812f991e1dd304a5af90"
+    uri: https://github.com/kudobuilder/kuttl/releases/download/v0.2.0/kuttl_0.2.0_linux_x86_64.tar.gz
+    sha256: "1b1c4f5be141c1469b445ad9ab9bf2e26261a37ce1af1b274839bec7bae8d306"
     bin: "./kubectl-kuttl"
+    files:
+    - from: "*"
+      to: "."
```
Comment (ahmetb) [inline plugins/kuttl.yaml:26]: this is redundant, can you delete?

**Comment by ahmetb** [comment]
`files: [from: *, to: .]` isn't necessary that's why we probably omitted that during the original code review.

If you use this https://krew.sigs.k8s.io/docs/developer-guide/release/automating-updates/ you can just make releases and this file will be updated by robots.

File: plugins/kuttl.yaml (hunk 1 above)
Comment (kensipe) [reply plugins/kuttl.yaml:26]: ah... we ran into this last time and fixed on the pr... but not in my script the gens this file... on it.. thx!

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

This pull request seems to straightforward.
I'll go ahead and accept it. :+1: Cheers.

/lgtm
/approve

**Pull request merged**